docs/developer.md

```diff
@@ -49,6 +49,13 @@ For more information about the [Remote container extension](https://code.visuals
 New code should be covered by basic unittests. Depending on the complexity of the feature, Reviewers may request more in-depth unittests.
 If necessary, the Freqtrade team can assist and give guidance with writing good tests (however please don't expect anyone to write the tests for you).
 
+#### How to run tests
+
+Use `py.test` in root folder to run all available testcases and confirm your local environment is setup correctly
```

Comment:
`py.test` is old terminology - `pytest` would be correct - and in the meantime [probably depecated](https://github.com/pytest-dev/pytest/issues/1629#issue-161422224).

I'm also not entirely sure if we need this here as well - this document does link to the [contributing guidelines](https://github.com/freqtrade/freqtrade/blob/develop/CONTRIBUTING.md) - which do have detailed instructions on [how to run pytest](https://github.com/freqtrade/freqtrade/blob/develop/CONTRIBUTING.md#before-sending-the-pr) with freqtrade.

Reply:
see `py.test` more as a standin for whatever is actually used. which I didn't know, because it's not documented anywhere in the docs, the first thing I was looking.

my experience today: I was trying to add a new feature, so I looked into the docs to see how tests are run for this project and I couldn't find it. so a simple, "hey this is how you get the ball rolling. having that hidden in another documents is not obvious. it would be better to just make a section. Tests: where are they, how to run them, what to expect and then link to the contributing guidelines for detailed instructions. But it should be stated somewhere, in the docs, how to tests this project from a clean fork

Reply:
That's great - as that's exactly how it is now if you read through the [developer setup](https://www.freqtrade.io/en/stable/developer/#developer-setup) section carefully.

it explains the tools that will be installed (assuming developers as audience -> audience knows how to use them, otherwise that's a research project on it's own).
it's then linking to the contributing guidelines for detailed instructions.